tools/windows-node-installer/pkg/cloudprovider/factory.go

```diff
@@ -1,7 +1,10 @@
 package cloudprovider
```

Comment:
You cannot add everything in cloudprovider directory, please add your provider(azure) specific directory